src/actor/Claimer.spec.ts

```diff
@@ -64,10 +65,12 @@ describe('Claimer', () => {
 
     blockchainApi.query.attestation.attestations.mockReturnValue(
       new Option(
+        registry,
         Tuple,
         new Tuple(
-          [H256, AccountId, Text, Bool],
-          [cType.hash, attester.getAddress(), undefined, false]
+          registry,
+          ['H256', AccountId, Text, Bool],
```

Comment:
The delegationNodeId is also a hash. We should probably use an optinal h256 for that?

```suggestion
          ['H256', AccountId, 'Option<H256>', Bool],
```

Reply:
Fixed in 4e9832e